src/Symfony/Bridge/Twig/TwigEngine.php

```diff
@@ -75,14 +75,14 @@ public function exists($name)
 
         $loader = $this->environment->getLoader();
 
-        if ($loader instanceof \Twig_ExistsLoaderInterface) {
+        if ($loader instanceof \Twig_ExistsLoaderInterface || method_exists($loader, 'exists')) {
```

Comment:
In Twig 2.0, the `exists()` method will be present in the `Twig_LoaderInterface`.